ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/SocialShare/SocialShare.html

```diff
@@ -0,0 +1,9 @@
+
+
+
+			<a id="swfMailTo" class="a-link-normal email shareText" title="Share via e-mail" href="">Share</a><span class="a-letter-space"></span>
```

Comment:
@uditnaagar777 remove the static text Share and replace it with ${properties.sharingLabel}